pkg/apis/serving/v1alpha1/servingruntime_types.go

```diff
@@ -254,6 +262,17 @@ type SupportedRuntime struct {
 	Spec ServingRuntimeSpec
 }
 
+// WorkerSpec is the schema for multi-node/multi-GPU feature
+// If WorkerSpec is specified, it is assumed that multi-node, multi-gpu will be used.
+// For now, it is specifically for vllm using ray
```

Comment:
We do not need to mention this in the spec, it is implementation details.

Reply:
Agree, I will remove it